examples/configurables/configurable.out

```diff
@@ -0,0 +1,11 @@
+# To run this sample, navigate to the directory that contains the
```

Comment:
The file name has to be the same as the directory. This will be ok if we rename the directory as "configurable".